SPECS/kmod-drbd.spec

```diff
@@ -0,0 +1,184 @@
+Name: kmod-drbd
+Summary: Kernel driver for DRBD
+Version: 9.2.10
+Release: 1.4%{?dist}
+
+# always require a suitable userland
+Requires: drbd-utils >= 9.27.0
+
+%global tarball_version %(echo "%{version}" | sed -e "s,%{?dist}$,," -e "s,~,-,")
+Source: http://oss.linbit.com/drbd/drbd-%{tarball_version}.tar.gz
+
+License: GPLv2+
+Group: System Environment/Kernel
+URL: http://www.drbd.org/
+BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-XXXXXX)
+
+BuildRequires: coccinelle
+BuildRequires: gcc
+BuildRequires: perl
+%if %{defined kernel_module_package_buildreqs}
+BuildRequires: %kernel_module_package_buildreqs
+%endif
+
+# XCP-ng patches
```

Comment:
No information on how this patch is obtained?

Reply:
After discussion:
- We must create a new DRBD repo on xcp-ng.
- Add a documentation regarding the open/close patch.
- Explain how to generate this patch.

Not required for this PR, the custom DRBD version is already released.